onnxruntime/python/tools/transformers/models/whisper/convert_to_onnx.py

```diff
@@ -183,22 +183,25 @@ def parse_arguments(argv=None):
         "--quantize_embedding_layer",
         required=False,
         action="store_true",
-        help="Produce beam search model with chained encdecinit and decoder.",
+        help="Quantize MatMul, GEMM, and Gather.",
```

Comment:
The change of whisper shall be in another PR.

Reply:
Fixed